app/src/main/java/com/github/h3lp3rs/h3lp/forum/data/ForumPostData.kt

```diff
@@ -15,7 +15,7 @@ import java.time.ZonedDateTime
 data class ForumPostData(
     val author: String,
     val content: String,
-    val postTime: ZonedDateTime,
+    val postTime: String,

```

Comment:
what is the  benefit of this change?

We lose some information with this. Someon posting message now in tawian will appeared to be send in the future for a person in the us 

The advantage of ZOnedDateTime is that it adapt the time with the zone the user is in